Atomic/scan.py

```diff
@@ -21,6 +25,7 @@ def cli(subparser):
     scanp.add_argument("--scan_type", default=None, help=_("define the intended scan type"))
     scanp.add_argument("--list", action='store_true', default=False, help=_("List available scanners"))
     scanp.add_argument("--scanner_args", default=None, help=_("Specify arguments to be passed to the scanner"))
+    scanp.add_argument("--harden", action='store_true', default=False, help=_("Create hardened image after scan"))
```

Comment:
I would like to talk this out next week when I return from PTO.  My preference would be that atomic scan remains neutral in this because it is all based on plugins.  For example, if i had a custom scanner, the --harden would not make any sense.  I have a couple of ideas on how to work around this and will propose them in IRC when I return.